src/APIServer/model/db.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://127.0.0.1/TWbackend').then(() => {
```

Comment:
Is there any way we can use the config file values rather than a hardcoded string?

Reply:
I will look into it in the next commit and combine it with the bin/www somehow. The hardcoding is a cludge, but I'd prefer to get this out and then looking into merging the /bin/www and config file or command line configs.

Reply:
No worries. If the file is autogenerated then we could be OK. Think we can leave it for now